WordPress/gutenberg
# Elements: Buttons: Fix the path with the css to style the cursor for all button elements

## What?

In this PR https://github.com/WordPress/gutenberg/pull/42102, we added a stylesheet to style .wp-elements-* elements, but those styles are not being loaded in the frontend.

## Why?

Because those styles are not being loaded in the frontend so they are not being applied.

## How?

In this PR I'm importing the `elements.scss` file in `common.scss` instead of `styles.scss`. Because `style.scss` is not loaded in the frontend and `common.scss` yes.

## Testing Instructions





1. Create a post
2. Add a search form to the post and publish it.
3. Navigate the public version of the post and check if the search form button has a `cursor:pointer` definition. If that's true, the styles are being applied correctly.


## Screenshots or screencast
